packages/components/src/Disclosure/Disclosure.css

```diff
@@ -0,0 +1,69 @@
+.arrowIconWrapper {
+  transform-origin: 50% 50% 0;
+  transition: transform var(--timing-base);
```

Comment:
Seeing this in practice I think `timing-quick` would be better for the arrow, I'm finding my eye gets pulled to it a bit too much 👁️